scripts/code_size_compare.py

```diff
@@ -629,23 +635,28 @@ def cal_sect_change(
             :param: sect: section to calculate from `size` tool. This could be
                           any instance variable in SizeEntry.
             :return: List of [section size of objects for new Git revision,
+                     section size of objects for old Git revision,
                      section size change of objects between two Git revisions]
             """
             if old_size and new_size:
                 new_attr = new_size.__dict__[sect]
-                delta = new_size.__dict__[sect] - old_size.__dict__[sect]
+                old_attr = old_size.__dict__[sect]
+                delta = new_attr - old_attr
                 change_attr = '{0:{1}}'.format(delta, '+' if delta else '')
             elif old_size:
                 new_attr = - old_size.__dict__[sect]
+                old_attr = old_size.__dict__[sect]
```

Comment:
I just notice that it is not correctly to set `new_attr = - old_attr` when `new_size` doesn't exsit. It should be consistent with the case that only `new_size` exists, e.g.
```C
if old_size and new_size:
    ...
elif old_size:
    new_attr = 'Removed'
    old_attr = old_size.__dict__[sect]
    delta = -old_attr
    change_attr = '{0:{1}}'.format(delta, '+' if delta else '')
elif new_size:
    new_attr = new_size.__dict__[sect]
    old_attr = 'NotCreated'
    delta = new_attr
    change_attr = '{0:{1}}'.format(delta, '+' if delta else '')
else:
    ...
```